lib/mindwendel_web/plugs/set_response_header_content_security_policy.ex

```diff
@@ -16,27 +16,33 @@ defmodule Mindwendel.Plugs.SetResponseHeaderContentSecurityPolicy do
   end
 
   def content_security_policy_directives() do
-    # Usually, the csp directive `connect-src 'self'` is enough,
-    # see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/connect-src .
-    #
-    # However, the `connect-src 'self'` does not resolve to websocket schemes in all browsers, e.g. Safari,
-    # see https://github.com/w3c/webappsec-csp/issues/7 .
-    #
-    # Therefore, we need to explicitly add the allowed websocket uris here so that live views will work in safari in combination with CSP policies.
-    # e.g. `connect-src ws://localhost:*` or `connect-src wss://#{@host}` .
-    "default-src 'none' ; " <>
+    [
+      "default-src 'none' ;",
+
+      # Usually, the csp directive `connect-src 'self'` is enough,
+      # see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/connect-src .
+      #
+      # However, the `connect-src 'self'` does not resolve to websocket schemes in all browsers, e.g. Safari,
+      # see https://github.com/w3c/webappsec-csp/issues/7 .
+      #
+      # Therefore, we need to explicitly add the allowed websocket uris here so that live views will work in Safari in combination with CSP policies.
+      # e.g. `connect-src ws://localhost:*` or `connect-src wss://#{@host}` .
       "connect-src 'self' #{
         %URI{
           scheme: get_websocket_scheme(),
           host: get_host(),
           port: get_port()
         }
-      } ; " <>
-      "font-src    'self' ; " <>
-      "frame-src   'self' ; " <>
-      "img-src     'self' data: ; " <>
-      "script-src  'self' 'unsafe-eval' ; " <>
-      "style-src   'self' 'unsafe-inline' ; "
+      } ;",
+      "font-src    'self' ;",
+      "frame-src   'self' ;",
+
+      # We add csp sources http: and https: to allow the browser to load the link preview image extracted from the idea body
+      "img-src     'self' data: https: http: ;",
+      "script-src  'self' 'unsafe-eval' ;",
```

Comment:
why did we need this again? sorry, I might have asked before and this stayed the same in this PR. But that's the only thing I find odd (again? ;-) ).

Reply:
I mean `unsafe-eval`